nengo/builder/operator.py

```diff
@@ -643,6 +643,70 @@ def step_dotinc():
         return step_dotinc
 
 
+class BsrDotInc(Operator):
```

Comment:
I think my stance on this depends on whether the optimizer is activated by default. If this is the case, the user would see that warning potentially for every model.